File: 5.ora/karaktercsere.c
#include <stdio.h>

char from[] = "aeiou";
char to[] = "eioua";

char csere(char c){
	int i=0;
	while(from[i]!='\0' && from[i]!=c){
		i++;
	}
	return from[i]!=c ? c : to[i];
}

int main(){
	unsigned char c, uj[256];
	int i;
	
	for(i=0; i<256; i++){
		uj[i] = i;
	}
	i=0;
	while(from[i]!='\0'){
		uj[from[i]] = to[i];
		i++;
	}
	
	
	while((c = getchar()) - 'A' > 0){
		if(c >= '0' && c <= '9') continue;
		putchar(uj[c]);
	}
	
	return 0;
}

File: 5.ora/karaktercsere.py
import sys
import string

uj = dict(zip("aeiou", "eioua"))
table = [uj.get(chr(i), chr(i)) for i in range(256)]

def csere(c):
	if c in uj:
		return uj[c]
	return c

for sor in sys.stdin:
	for c in sor:
		print(table[ord(c)], end="")

